lib/docusign_rest/client.rb

```diff
@@ -426,6 +426,22 @@ def get_tabs(tabs, options, index)
       tab_array
     end
 
+    # Allows adding extra attributes to the file array
+    def create_file_ios_with_params(files)
+      ios = []
```

Comment:
I agree, and there's a lot of thing's we'd like to do differently in this Gem. This is definitely just a few lines of hacky code to prevent us from having to write our own Docusign API client.